Ejercicio GIT

ijchavez opened the pull request:
Se adjunta el ejercicio GIT propuesto en la clase 12. El mismo arranca en la linea 64 y tiene la siguiente consigna:
Ejercicio
Crear un método llamado método LoginFailureTest
En el sitio de salesforce: "https://login.salesforce.com/?locale=eu"
Validar que se encuentre el logo en el sitio (utilizar un WebElement)
Completar el username con “test@test.com”
Completar el campo Password con “123466”
Hacer click en Login
Imprimir en pantalla el mensaje de error
El orden de prioridad de este test, debe ser 3

ijchavez committed:
Ejercicio GIT

Se adjunta el ejercicio GIT propuesto en la clase 12. El mismo arranca en la linea 64 y tiene la siguiente consigna:
Ejercicio
Crear un método llamado método LoginFailureTest
En el sitio de salesforce: "https://login.salesforce.com/?locale=eu"
Validar que se encuentre el logo en el sitio (utilizar un WebElement)
Completar el username con “test@test.com”
Completar el campo Password con “123466”
Hacer click en Login
Imprimir en pantalla el mensaje de error
El orden de prioridad de este test, debe ser 3

seleniumcurso commented:
Muy buen trabajo!!

seleniumcurso commented on testngSalesforce.java:
```diff
@@ -0,0 +1,104 @@
+    }
+
+    @Test (priority = 2)/*(enabled = false)*/
+    public void FooterIsValid() throws InterruptedException {
```
el nombre de los metodos debe empezar en minusculas y es recomendable que tenga la palabra test al final

ijchavez replied:
Cambiado a:
FooterIsValidTest
RememberMeIsSelectedTest

seleniumcurso replied:
Esta pusheado los cambios?

ijchavez replied:
Si, el tema es que al poner el nombre de la clase en mayúscula, se creo un nuevo archivo .java que quedó mas arriba. IntelliJ me obliga al cambiar el nombre de la clase a cambiar el .java tambien haciendo refactor>> rename
![Selección_009](https://user-images.githubusercontent.com/72580524/103485503-90582280-4dd5-11eb-967e-37e6140a9d04.png)
![Selección_010](https://user-images.githubusercontent.com/72580524/103485504-90f0b900-4dd5-11eb-8312-9ca2901e0075.png)

seleniumcurso commented on testngSalesforce.java:
```diff
@@ -0,0 +1,104 @@
+    @Test(priority = 3)
+    public void LoginFailureTest() throws InterruptedException {
+        //Uso una nueva URL porque cambia una parte con respecto a https://login.salesforce.com/
+        String nuevaURL = "https://login.salesforce.com/?locale=eu";
```
podrias definir una variable estatica con esta url

ijchavez replied:
De hecho estaba en el ejercicio y la pase por alto. Cree 2:
    public static final String SALESFORCE_URL = "https://login.salesforce.com/";
    public static final String SALESFORCELOCAL_URL = "https://login.salesforce.com/?locale=eu";

seleniumcurso commented on testngSalesforce.java:
```diff
@@ -0,0 +1,104 @@
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+public class testngSalesforce {
```
el nombre de las clases siempre empieza en mayusculas

ijchavez replied:
Cambiado a TestngSalesforce

ijchavez commented:
Gracias por tus comentarios y por tu reconocimiento Emiliano. Ahí hice los cambios y ya los estoy subiendo.

Gracias por tu tiempo ademas.

ijchavez committed:
Cambios:

Se agrega:
Cambio de nombre de metodos a:
- FooterIsValidTest
- RememberMeIsSelectedTest

Creación de dos variables estáticas:
public static final String SALESFORCE_URL = "https://login.salesforce.com/";
public static final String SALESFORCELOCAL_URL = "https://login.salesforce.com/?locale=eu";

Cambio de nombre de la clase a:
Cambiado a TestngSalesforce

ijchavez committed:
Update TestngSalesforce.java

nombres de los metodos en minuscula
